src/lemlib/devices/encoder/motor.cpp

```diff
@@ -51,7 +54,7 @@ float MotorEncoder::getAngle() {
  * Reset the motor encoders.
  */
 bool MotorEncoder::reset() {
-    lastAngle = 0;
+    lastAngle = 0_deg;
```

Comment:
```suggestion
    lastAngle = 0_rad;
```
change to radians for consistency